doc/source/ray-air/experimental-features.rst

```diff
@@ -0,0 +1,128 @@
+.. _air-experimental-features:
+
+================================
+Experimental features in Ray AIR
+================================
+
+We're testing a number of experimental features in Ray AIR.
+
+During development and internal dogfooding, the features
+are disabled per default. You can opt-in by setting an
+environment variable.
+
+After some time, we enable the feature per default to gather
+more feedback from the community. In that case, they can still
+be disabled using the same environment variable. This will
+fully revert to the old behavior, should the new behavior
+not suffice your needs.
+
+If you run into issues with experimental features,
+please `open an issue <https://github.com/ray-project/ray/issues/>`_
+on GitHub and let us know! This will give
+us the chance to incorporate your feedback before we remove
+the old implementation and make the new implementation the
+default.
+
+.. note::
+
+    Experimental features can undergo frequent changes,
+    especially on the master branch and the nightly wheels.
+
+.. _air-experimental-new-output:
+
+New output engine
+-----------------
+
+.. note::
+
+    This feature is *enabled per default* starting Ray 2.5.
+
+    To disable, set the environment variable ``RAY_AIR_NEW_OUTPUT=0``.
+
+We've added a new output engine for Ray Train and Ray Tune runs.
+
+The new output engine will affect how the training progress
+is printed in the console.
+
+The new features include:
+
+- Ray Train runs will report status relevant to the single training run.
+  It will not use the default Ray Tune table layout from before.
+- The table format has been updated.
+- We report configurations and observed metrics in a different format
+- The default metrics displayed for e.g. RLlib runs are heavily reduce
+  in the console output.
+- We generally uncluttered a lot of output and made it easier to
+  read.
+
+The new output currently only works for the regular console.
+Notably, it is automatically disabled when Jupyter Notebooks
+or Ray client is used.
+
+
+.. _air-experimental-rich:
+
+Sticky table layout
```

Comment:
can we comment on how this relates to the above output engine?

Reply:
Is the first sentence enough?

> As part of the :ref:`new output engine <air-experimental-new-output>`,
we've introduced a sticky table layout.

Reply:
ok, thanks